pro_gan_pytorch/CustomLayers.py

```diff
@@ -19,15 +19,16 @@ def __init__(self, c_in, c_out, k_size, stride=1, pad=0, initializer='kaiming',
         :param bias: whether to use bias or not
         """
         super(_equalized_conv2d, self).__init__()
-        self.conv = th.nn.Conv2d(c_in, c_out, k_size, stride, pad, bias=True)
+        self.conv = th.nn.Conv2d(c_in, c_out, k_size, stride, pad, bias=False)
```

Comment:
Ah right! It was set to `True` here and in `Linear`. In `deconv` it was already false. Thanks for the fix! :+1: